### PR TITLE
Feature/audiobook

### PR DESCRIPTION
**Pull request by Claeb1mb**
Audiobook boilerplate files & folder created.

### CODE REVIEW

**Comment by Antashma** [comment]
looks good

**Review by Antashma — DISMISSED** [review]

**Review by kavya3v — DISMISSED** [review]
Approved, please proceed . Note: To clean up import in RenderAudioBook

File: src/components/pages/AudioBook/AudioBookContainer.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import RenderAudioBook from './RenderAudioBook';
+
+function AudioBook(...props) {
```
Comment (sunkist5691) [inline src/components/pages/AudioBook/AudioBookContainer.js]: Try to destructure instead of using only props.

Comment (sunkist5691) [reply src/components/pages/AudioBook/AudioBookContainer.js]: { LoadingComponent, ...props }

**Review by sunkist5691 — CHANGES_REQUESTED** [review]

**Review by Claeb1mb — COMMENTED** [review]
changes made thanks Joon

**Review by sunkist5691 — APPROVED** [review]

**Review by kavya3v — APPROVED** [review]
Destructure looks fine

**Review by Antashma — APPROVED** [review]
👍

**Pull request merged**